taxi/forms.py

```diff
@@ -0,0 +1,60 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+from django.contrib.auth import get_user_model
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    @staticmethod
+    def validate_license_number(license_number):
+        if (
+            license_number
+            and len(license_number) == 8
+            and license_number[:3].isalpha()
+            and (license_number[:3].isupper() and license_number[3:].isdigit())
+        ):
+            return True
+        else:
+            return False
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if self.validate_license_number(license_number):
+            return license_number
+        else:
+            raise ValidationError("This license is wrong")
```

Comment:
```suggestion
    @staticmethod
    def validate_license_number(license_number):
        if (
            license_number
            and len(license_number) == 8
            and license_number[:3].isalpha()
            and (license_number[:3].isupper() and license_number[3:].isdigit())
        ):
            return True
        return False

    def clean_license_number(self):
        license_number = self.cleaned_data["license_number"]
        if self.validate_license_number(license_number):
            return license_number
        raise ValidationError("This license is wrong")
```
`else` redundant in this case